screens/Settings/notificationHandler.ts

```diff
@@ -0,0 +1,76 @@
+import * as Notifications from 'expo-notifications'
```

Comment:
This file should be moved to the root directory under a folder called ./notifications/ as it could be used throughout the app in the future not just for settings.

Reply:
> Nice work @SalikSayyed! I will review the code soon. In terms of the UI, what about placing the weekday / time part inside a modal that pops up instead of placing it inside the list? You can use `react-native-paper`'s `Dialog` component for the modal and put it in its own component like `NotificationDialog` :) This will make things easier to move around in the future.
> 
> https://callstack.github.io/react-native-paper/dialog.html

will do soon